_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -32011,11 +32008,11 @@
 	},
 /obj/structure/cable,
 /obj/machinery/door/window{
+	atom_integrity = 300;
 	base_state = "rightsecure";
 	dir = 1;
 	icon_state = "rightsecure";
 	name = "Primary AI Core Access";
-	atom_integrity = 300;
 	req_access_txt = "16"
 	},
```

Comment:
why did this line, over on the AI sat, change in this PR?

Reply:
I have no idea, but I didn't touch the ai sat in any way, didn't even go near it. Swearsies!